packages/ui/src/App.tsx

```diff
@@ -10,30 +10,33 @@ import { ThemeProvider as MuiThemeProvider } from '@mui/material/styles'
 import { AccountNamesContextProvider } from "./contexts/AccountNamesContext"
 import { NetworkContextProvider } from "./contexts/NetworkContext"
 import MainLayout from "./components/layout/Main";
+import SnackbarContextProvider from "./components/SnackBar/SnackStackProvider"
 
 
 const App = () => {
   const queryClient = new QueryClient()
 
   return (
     <MuiThemeProvider theme={theme}>
-      <ToastContextProvider>
-        <BrowserRouter>
-          <NetworkContextProvider>
-            <QueryClientProvider client={queryClient}>
-              <ApiContextProvider>
-                <AccountContextProvider>
-                  <AccountNamesContextProvider>
-                    <MultiProxyContextProvider>
-                      <MainLayout />
-                    </MultiProxyContextProvider>
-                  </AccountNamesContextProvider>
-                </AccountContextProvider>
-              </ApiContextProvider>
-            </QueryClientProvider>
-          </NetworkContextProvider>
-        </BrowserRouter>
-      </ToastContextProvider>
+      <SnackbarContextProvider>
+        <ToastContextProvider>
```

Comment:
please remove the old `ToastContextProvider` altogether, it's no needed any more :)